types/p2p.yaml

```diff
@@ -8,16 +8,13 @@ NetworkIdentity:
     p2p_addresses:
       type: array
       items:
-        allOf:
-          - $ref: "./p2p.yaml#/Multiaddr"
-          - description: "Node's addresses on which eth2 RPC requests are served. [Read more](https://docs.libp2p.io/reference/glossary/#multiaddr)"
+        $ref: "./p2p.yaml#/Multiaddr"
+        description: "Node's addresses on which eth2 RPC requests are served. [Read more](https://docs.libp2p.io/reference/glossary/#multiaddr)"
     discovery_addresses:
       type: array
       items:
-        allOf:
-          - $ref: "./p2p.yaml#/Multiaddr"
-          - description: "Node's addresses on which is listening for discv5 requests. [Read more](https://docs.libp2p.io/reference/glossary/#multiaddr)"
-          - example: "/ip4/7.7.7.7/udp/30303/p2p/QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N"
```

Comment:
This example is kinda odd in the first place as is uses the default EL p2p port. Maybe the intend was to use a udp example here?